### PR TITLE
Add documentation for the Bright Sky weather entity

### PR DESCRIPTION
**Pull request by mweinelt**
## Proposed change

Add documentation for the bright sky weather entity.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/37800
- Link to parent pull request in the Brands repository: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/weather.brightsky.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - Weather
+ha_release: 0.114
+ha_iot_class: Cloud Polling
+ha_quality_scale: silver
+ha_codeowners:
+  - '@mweinelt'
+ha_domain: brightsky
+---
+
+The `brightsky` platform uses the [Bright Sky](https://darksky.net/) JSON API to query meteorological provided by the Deutsche Wetterdienst (DWD)
```
Comment (Mic92) [inline source/_integrations/weather.brightsky.markdown]: Wrong url?

```suggestion
The `brightsky` platform uses the [Bright Sky](https://brightsky.dev) JSON API to query meteorological provided by the Deutsche Wetterdienst (DWD)
```

File: source/_integrations/weather.brightsky.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Bright Sky"
+description: "Instructions on how to integrate Bright Sky within Home Assistant."
+logo: dwd.png
```
Comment (klaasnicolaas) [inline source/_integrations/weather.brightsky.markdown]: Please add the logo to our [Brands repository](https://github.com/home-assistant/brands)

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Please rename the file from `weather.brightsky.markdown` to `brightsky.markdown`

File: source/_integrations/brightsky.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+## Configuration
+
+The API currently imposes no rate limits, this platform updates at most every 10 minutes.
+
+To add Bright Sky to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+weather:
+  - platform: brightsky
```
Comment (frenck) [inline source/_integrations/brightsky.markdown]: We no longer accept platform integrations. Furthermore, integrations that are connecting to a device or service need to have a config flow in the UI.

For more information, see my comment on the parent PR.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed by the contributor.

**Pull request closed without merging**